File: Python/11.py
def IsTriangleValid(a, b, c):
    if a + b > c and b + c > a and a + c > b:
        return True
    return False

print("Enter the sides of the triangle: ")
a = int(input())
b = int(input())
c = int(input())

if(IsTriangleValid(a, b, c)):
    print("YES")
else:
    print("NO")

File: C++/11.cpp
#include <iostream>

using namespace std;

bool IsTriangleValid(double a, double b, double c)
{
    return (a + b > c) && (a + c > b) && (b + c > a);
}

int main()
{
    double a, b, c;
    cout << "Enter the lengths of the three sides of the triangle: ";
    cin >> a >> b >> c;

    if (IsTriangleValid(a, b, c))
    {
        cout << "YES" << endl;
    }
    else
    {
        cout << "NO" << endl;
    }
}
